tensorflow_estimator/tools/pip_package/setup.py

```diff
@@ -30,7 +30,7 @@
 # This version string is semver compatible, but incompatible with pip.
 # For pip, we will remove all '-' characters from this string, and use the
 # result for pip.
-_VERSION = '2.7.0'
+_VERSION = '2.8.0'
```

Comment:
This is on the 2.7 branch. It should be 2.7

Reply:
hi please review..made the changes